architecture/design/docdb-automatic-tablet-splitting.md

```diff
@@ -63,65 +69,65 @@ When leader tablet server receives `SplitTablet` RPC it adds a special Raft reco
   - This replica joins the cluster back after follower_unavailable_considered_failed_sec. In this case replica is considered as failed and is evicted from the Raft group, so we don’t need to hold the old tablet anymore.
 
-  Note: by default `follower_unavailable_considered_failed_sec` = `log_min_seconds_to_retain`, but these flags could be adjusted.
+  Note: by default `follower_unavailable_considered_failed_sec` = `log_min_seconds_to_retain`, but these flags can be adjusted.
 
 
 ### Document Storage Layer splitting
-- We copy the RocksDB to additional directory using hard links and add metadata saying that only 
+- We copy the RocksDB to additional directory using hard links and add metadata saying that only
 part of the key range is visible. Remote bootstrap will work right away.
-Next major compaction will remove all key-value pairs which are no longer related to the tablet for new tablets due to 
-split. Also later we can implement cutting of RocksDB without full compaction (see below in this section).
-- We store tablet's key bounds inside `KvStoreInfo` tablet metadata. `BoundedRocksDbIterator` is filtering out non relevant keys. We can 
-also propagate key boundaries to regular RocksDB instance so it has knowledge about non relevant keys and we can 
+Next major compaction will remove all key-value pairs which are no longer related to the new tablets due to
+split. Later, we can implement cutting of RocksDB without full compaction (see below in this section).
+- We store tablet's key bounds inside `KvStoreInfo` tablet metadata. `BoundedRocksDbIterator` filters out non relevant keys. We can
+also propagate key boundaries to a regular RocksDB instance so that it has knowledge about non relevant keys, and we can
 implement RocksDB optimizations like truncating SST files quickly even before the full compactions.
-- **Performance note: remote bootstrap could download not relevant data from new tablet if remote bootstrap is happening 
+- **Performance note: remote bootstrap could download irrelevant data from new tablet if remote bootstrap is happening
 before compaction.**
 
 - Snapshots (created by `Tablet::CreateSnapshot`)
-  - The simplest solution is to clone snapshot as well using hard-link, but snapshots are immutable and once they will 
-  be re-distributed across different tservers it will increase space amplification. Snapshots are currently only used 
-  for backup and are quickly deleted. But it would be good to implement cutting of RocksDB without full compaction - 
-  could be done later. 
+  - The simplest solution is to clone snapshot as well using hard-link, but snapshots are immutable and once they will
+  be re-distributed across different tservers it will increase space amplification. Snapshots are currently only used
+  for backup and are quickly deleted. But it would be good to implement cutting of RocksDB without full compaction -
+  could be done later.
```

Comment:
Had some trouble understanding this part.  Should it be

"The simplest solution is to clone snapshots as well using hard-link _because_ snapshots are immutable"?

Reply:
I think this part is outdated and the current state is:
"
Tablet data could also contain snapshots (created by Tablet::CreateSnapshot).
We don’t need to copy snapshots from the parent tablet to post-split tablets, but we might need to keep them for the parent tablet even after the split is completed.
"